manifest.yaml

```diff
@@ -1,4 +1,4 @@
-name: teracy-dev-v05-migration
+name: teracy-dev-v05-compat
 version: "0.1.0-SNAPSHOT"
-description: migration extension from teracy-dev v0.5 yo v0.6
-target: ">= 0.6.0-a1-SNAPSHOT" # the teracy's target version (optional, not specified means all)
+description: migration extension from teracy-dev v0.5 to v0.6
+target: ">= 0.6.0-a5-SNAPSHOT" # the teracy's target version (optional, not specified means all)
```

Comment:
">= 0.6.0-a4" should work, keep support as low as possible (so that we can support more versions), should use the released version instead of the current SNAPSHOT version.

Reply:
do we had extension path in a4?

Reply:
it's on teracy-dev-core v0.4.0-SNAPSHOT, not from the teracy-dev kernel.

https://github.com/teracyhq/dev/issues/498 will help, you can create dependencies for now so it should work later.